hoomd/md/NeighborListGPUBinned.cu

```diff
@@ -217,7 +214,7 @@ __global__ void gpu_compute_nlist_binned_kernel(unsigned int* d_nlist,
             if (!use_index)
                 cur_tdb = d_cell_tdb[cli(cur_offset, neigh_cell)];
             else
-                cur_tdb = make_scalar4(postype_j.w, d_diameter[j], __int_as_scalar(d_body[j]), 0);
```

Comment:
Is this 0 just to prevent more refactoring?

Reply:
Yes, the cell list (optionally) repacks the particle type, diameter and body ("tdb") in a memory-efficient layout for `NeighborListBinned`.

Now that I look at it, `NeighborListBinned` is the only consumer of this data structure, so I should refactor `CellList` to store only type and body on this branch, removing this 0.